.gitignore

```diff
@@ -3,6 +3,9 @@ work
 .DS_Store
 .flattened-pom.xml
 
+**/generated
```

Comment:
Is this about `build/generated`? If yes, it's already covered by the `build/` being ignored

Reply:
No, it's about generated source in module directories. They appear after compilation using Gradle